### PR TITLE
Adds new outline README

### PR DESCRIPTION
**Pull request by sleepypioneer**
- [x] Update README with new outline for tutorial
- [ ] Remove pubnub files
- [ ] Update Introduction to working with hardware & Set up instructions
- [ ] Add page with items to buy (if not in live session)
- [ ] Update setting up a sensor & collecting data (code comments/ tests)
- [ ] Update storing Data with InduxDB
- [ ] Update visualizing Data with Grafana
- [ ] Update continuously collecting Data, Supervisor
- [ ] Add conclusion, further reading, inspiration links

### CODE REVIEW

**Comment by sleepypioneer** [comment]
**WORK IN PROGRESS**

File: README.md [hunk 1]
```diff
@@ -1,31 +1,94 @@
...
+-Micro SD card with 4-16 GB
+-Micro SD to usb adapter
+-Circuit breadboard
+-DHT22 sensor from Adafruit with 10ohm resistor
+-3 connecting wires
 
-### Part 2
+If you are not attending one of our live sessions where we have recieved sponsorship for hardware you can find these items to purchase online, here is a list of places you may find [them](hardware-shopping-list.md)
+
+For setting up the raspberry Pi follow the instructions in [0_pi-setup](pi-setup/initial-setup.md)
```
Comment (bwnstck) [inline README.md:74]:
```suggestion
For setting up the raspberry Pi follow the instructions in [0_pi-setup](0_pi-setup/initial-setup.md)
```
some links are missing the trailing number

File: README.md [hunk 2]
```diff
@@ -1,31 +1,94 @@
...
+
+### Setting up a sensor & collecting data
+
+For setting up the sensor follow the instructions in [1_sensor-setup](sensor-setup/humidity-sensor-setup.md)
```
Comment (bwnstck) [inline README.md:78]:
```suggestion
For setting up the sensor follow the instructions in [1_sensor-setup](1_sensor-setup/humidity-sensor-setup.md)
```

**Review by bwnstck — CHANGES_REQUESTED** [review]
Fixes some links

**Review by terezaif — APPROVED** [review]

**Pull request merged**